packages/visx-xychart/src/components/series/private/BaseAreaSeries.tsx

```diff
@@ -26,23 +27,31 @@ export type BaseAreaSeriesProps<
   lineProps?: Omit<LinePathProps<Datum>, 'data' | 'x' | 'y' | 'children' | 'defined'>;
   /** Rendered component which is passed path props by BaseAreaSeries after processing. */
   PathComponent?: React.FC<Omit<React.SVGProps<SVGPathElement>, 'ref'>> | 'path';
-} & Omit<React.SVGProps<SVGPathElement>, 'x' | 'y' | 'x0' | 'x1' | 'y0' | 'y1' | 'ref'>;
+} & Omit<
+    React.SVGProps<SVGPathElement>,
+    'x' | 'y' | 'x0' | 'x1' | 'y0' | 'y1' | 'ref' | 'pointerEvents'
+  >;
+
+const eventSourceSubscriptions = [AREASERIES_EVENT_SOURCE, XYCHART_EVENT_SOURCE];
```

Comment:
I think actually `eventSourceSubscriptions` must be equal to `${SERIESTYPE_EVENT_SOURCE}-${dataKey}`, otherwise Series could trigger events for other Series of the same type.